echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/monitor/PubsubEventMonitorSpec.groovy

```diff
@@ -108,26 +108,6 @@ class PubsubEventMonitorSpec extends Specification implements RetrofitStubs {
     // TODO(jacobkiefer): Add Kafka cases when that is implemented.
   }
 
-  @Unroll
```

Comment:
Meant to comment on this before. This logic is already tested here https://github.com/spinnaker/echo/pull/224/files#diff-167c2efbfefee403c1b06a329d235fd0R143.